src/AppBundle/Controller/IdeaController.php

```diff
@@ -33,38 +33,18 @@ public function viewAction($id)
         /** @var IdeaService $ideaService */
         $ideaService = $this->get('idea.service');
 
+        $idea = $ideaService->getIdea($id);
+
         return $this->render(
             'AppBundle:Idea:view.html.twig',
             [
-                //'idea' => $ideaService->getIdea($id),
-                // Static data to prettify the idea page
                 'idea' => [
-                    'id' => '123',
-                    'title' => 'What about movie',
-                    'createdAt' => time(),
-                    'description' => 'On Christmas Eve, New York City Police Detective Lieutenant John McClane arrives
-                    in Los Angeles to reconcile with his estranged wife, Holly. McClane is driven to the Nakatomi Plaza
-                    building for a company Christmas party by Argyle. While McClane changes clothes, the party is
-                    disrupted by the arrival of Hans Gruber and his heavily armed group: Karl, Franco, Tony, Theo,
-                    Alexander, Marco, Kristoff, Eddie, Uli, Heinrich, Fritz, and James.',
+                    'id' => $idea->getId(),
+                    'title' => $idea->getTitle(),
+                    'createdAt' => $idea->getCreatedAt(),
+                    'description' => $idea->getDescription(),
                     'user' => [
-                        'username' => 'Bruce Willis'
-                    ],
-                    'comments' => [
-                        [
-                            'user' => [
-                                'username' => 'David Guetta',
-                            ],
-                            'content' => 'Great!',
-                            'createdAt' => time(),
-                        ],
-                        [
-                            'user' => [
-                                'username' => 'Armin van Buuren',
-                            ],
-                            'content' => 'Like it!',
-                            'createdAt' => time(),
-                        ]
+                        'username' => $idea->getUser(),
```

Comment:
Does `$idea->getUser()` return an instance of the `User` model? If so, you might want to fetch the data from the user first, and set the 'final' data (more primitive types) as template variables.

If you use the following in your template `{{ user.username }}`, you're implicitly using a getter method in your view, which makes refactoring harder later on (since you would also have to 'fix' all your views)